src/main/java/duckling/Server.java

```diff
@@ -1,11 +1,101 @@
 package duckling;
 
+import duckling.requests.Request;
+import duckling.requests.RequestBuilder;
+import duckling.responders.Responders;
+
+import java.io.*;
+import java.net.*;
+import java.util.concurrent.*;
+
 public class Server {
+    public static final String CLRF = "\r\n";
```

Comment:
Ah, okay!  So maybe not necessary now, but if I need it, an entire `*Constants` object isn't necessarily a bad thing (or a frowned-upon thing)?